lib/raygun.batch.ts

```diff
@@ -56,7 +56,7 @@ export class RaygunBatchTransport {
 
   stopProcessing() {
     if (this.timerId) {
-      debug("batch transport - stopping");
+      debug(`[raygun.batch.ts] Batch transport - stopping`);
```

Comment:
The reason why the `debug` calls don't have an addtional `[Raygun4Node]` tag prefix is that they are used for different purposes in the library and get output via a different mechanism. 

I don't know the history of this choice, and maybe it should be revisited in the future (as in : Why do we use `debug` as an external library as well as `console.log/.warn etc ?). 

Anyway - Using `debug` automatically internally tags debug statements into a `raygun` context and putting `[Raygun4Node]` in there as well would have not been the right choice.

Reply:
If this is the package - https://www.npmjs.com/package/debug
It seems to do a lot of things but still not sure what benefit we get out of the package vs the just the regular `debug`

Reply:
This seems related https://github.com/MindscapeHQ/raygun4node/commit/10dba8ae6547c8f89d9829cf870c813bcde4d414

Reply:
Also what ChatGPT spits out is - 

> The `debug` npm package and the `debug` utility/module are essentially the same thing, but they're used in different contexts.
> 
> 1. **npm Package (`debug`)**: The `debug` npm package is used in Node.js applications to add debug logging functionality. It provides a simple and flexible way to log debug messages to the console or other destinations based on environment variables. You can install it in your Node.js projects via npm (`npm install debug`) and use it to instrument your code with debug statements. The npm package allows you to define debug namespaces and enable/disable them dynamically at runtime.
> 
> 2. **Utility/Module (`debug`)**: The `debug` utility/module is often used in browser development environments and JavaScript frameworks like Vue.js and React. It's a lightweight debugging tool that you can include in your code to output debug messages to the console. This `debug` module is typically included directly in your JavaScript code, rather than being installed via npm. It's commonly used during development to log information about the application's state, variables, or behavior for debugging purposes.
> 
> In summary, the `debug` npm package and the `debug` utility/module serve the same purpose: they provide a convenient way to add debug logging to your JavaScript/Node.js applications. The npm package is typically used in Node.js environments, while the utility/module is used in browser-based development or JavaScript frameworks. They both offer similar functionality for logging debug messages, but they're used in different contexts depending on the platform or environment of your application.

Reply:
So as you mentioned Kai, the only thing that would be the difference is the library serving a different purpose in some scenarios